bundles/best_practices.rst

```diff
@@ -68,30 +68,31 @@ The basic directory structure of an AcmeBlogBundle must read as follows:
 .. code-block:: text
 
     <your-bundle>/
-    ├─ AcmeBlogBundle.php
-    ├─ Controller/
+    ├─ src/
+    |   ├─ AcmeBlogBundle.php
+    |   ├─ Controller/
+    |   ├─ Resources/
+    |   |   ├─ config/
+    |   |   ├─ doc/
```

Comment:
I think `doc` can be moved to the root and `docs` seems more common https://github.com/symfony/symfony-docs/issues/5691#issuecomment-385971096